### ISSUE TITLE
Release for libsbml-experimental-5.18.1

### ISSUE THREAD

**Issue by matthiaskoenig** [libsbmljs/libsbmljs#2]
Would it be possible to generate a release for the latest libsbml-experimental, i.e., 5.18.1?
I mainly would need the updated distrib code. 

Best Matthias

**Comment by 0u812** [comment]
Many thanks for you interest in this project, Matthias. I will update the experimental build to 5.18.1. The libsbmljs wrapper needs a small update for this to work but I don't think it will be difficult.

**Comment by matthiaskoenig** [comment]
@0u812 Thanks for this project. This solves so many issues I had using SBML in a web context.

**Comment by 0u812** [comment]
No problem. I uploaded 5.18.1-alpha1 based on libsbml-experimental svn revision `26089`. I had to disable the distrib and spatial packages because they both have changes, and the distrib wrapper in particular needs to be rewritten for libsbmljs. I will get to that as soon as I can. I believe the following npm command should install the version I just uploaded:

```
npm install --save libsbmljs_experimental@5.18.1-alpha1
```

Please let me know if you run into any trouble using it.

**Comment by matthiaskoenig** [comment]
I don't need spatial at all. But distrib would be very nice.
I tried to run the examples and everything works when I do the example on the console with
```
npx babel-node --presets @babel/env index.js
```
I then tried to run the simple model creation example from within a vue.js app. Here I get the following errors in chrome and firefox and cannot create the SBML. Not sure what is wrong here.
```
webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8 wasm streaming compile failed: TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
(anonymous) @ webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8
webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8 falling back to ArrayBuffer instantiation
(anonymous) @ webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8
webpack-internal:///./node_modules/vue/dist/vue.runtime.esm.js:8412 Download the Vue Devtools extension for a better development experience:
https://github.com/vuejs/vue-devtools
webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8 failed to asynchronously prepare wasm: CompileError: WebAssembly.compile(): expected magic word 00 61 73 6d, found 3c 21 44 4f @+0
(anonymous) @ webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8
webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8 CompileError: WebAssembly.compile(): expected magic word 00 61 73 6d, found 3c 21 44 4f @+0
webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8 CompileError: WebAssembly.compile(): expected magic word 00 61 73 6d, found 3c 21 44 4f @+0
abort @ webpack-internal:///./node_modules/libsbmljs_stable/libsbml.js:8
localhost/:1 Uncaught (in promise) abort({}). Build with -s ASSERTIONS=1 for more info.
```

**Comment by 0u812** [comment]
Converting the hex values `3c 21 44 4f` to ascii yields `<!DO`, which is the start of a `<!DOCTYPE>` declaration. It looks like your code is somehow being served an HTML file instead of the correct .wasm file. It's hard to say what might be causing this. Is your code available? I think I can rule out the 5.18.1-alpha1 package itself being the source because it works with my [minimal Webpack example](https://github.com/libsbmljs/webpack-example) when I plug in the new package name and version in the appropriate places.

**Comment by matthiaskoenig** [comment]
yes, I don't think it is related to the 5.18.1 version, is the context I am using libsbml.js in. Will try to figure out what is going on. 

Yes, minimal examples are working with the new 5.18.1-alpha.
